browser/main/Detail/MarkdownNoteDetail.js

```diff
@@ -25,15 +25,23 @@ class MarkdownNoteDetail extends React.Component {
       note: Object.assign({
         title: '',
         content: ''
-      }, props.note)
+      }, props.note),
+      editorStatus: false,
+      isLocked: false
     }
     this.dispatchTimer = null
+
+    this.showLockButton = () => this.handleShowLockButton()
```

Comment:
Please use `Function.prototype.bind`. It is more clearly which `this` is assigned.

```javascript
this.showLockButton = this.showLockButton.bind(this);
```